weave-js/src/components/Panel2/PanelPlot/PanelPlot2Inner.tsx

```diff
@@ -110,6 +111,33 @@ const PLOT_TEMPLATE: VisualizationSpec = {
   },
 };
 
+function tooltipNoCache(
+  row: any,
+  s: SeriesConfig,
+  flatResultNode: Node,
+  weave: Weave
```

Comment:
Could the last parameter just be `opStore: OpStore` - that seems like all you need.

Reply:
done